modules/network-observability-viewing-network-events.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-viewing-network-events_{context}"]
+= Viewing network events
+You can view network events, such as network flows that are dropped or allowed by a network policy, by editing the `FlowCollector` to the specifications in the following YAML example.
+
+.Prerequisites
+* Must have the `OVNObservability` specified in the `FeatureGate` custom resource. For more information, see "Enabling features using feature gates" in the "Additional resources" of this section.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. In the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster*, and then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource to enable viewing `NetworkEvents`, for example:
++
+[id="network-observability-flowcollector-configuring-networkevents{context}"]
+.Example `FlowCollector` configuration
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+   agent:
+    type: eBPF
+    ebpf:
+      sampling: 1       <1>
+      privileged: true  <2>
+      features:
+       - "NetworkEvents"
+----
+<1> The `sampling` parameter is set to a value of 1 so that all network events are captured.
+<2> The `privileged` parameter is set to `true` because the `OVN observability` library needs to access local OVS socket and OVN databases
+
+.Verification
+. Navigate to the *Network Traffic* view and select the *Traffic flows* table. 
+. You should see the new column, *Network Events*, where you can view information about impacts that network policies, admin policies, and egress firewalls have on network flows.
++
+.Examples of Network Events output
+[source,text]
+----
+Dropped by cluster multicast policy, direction Ingress
+----
++
+[source,text]
+----
+Allowed by network policy iperf.iperf3-server-access-egress, direction Egress
+----
++
+[source,text]
+----
+Allowed by admin network policy allow-egress-iperf, direction Egress
+----
```

Comment:
How I can view the above examples are their link to other doc there is todo ?

Reply:
@msherif1234 sorry I thought I posted these but I accidentally started a review instead of adding a single comment. I thought these examples are printed in the *Network Events* column? You shared this screencapture with me a while ago: 
![image](https://github.com/user-attachments/assets/624d18df-1ca0-4aac-8c3b-6216fff8f4e2)